Packs/PrismaCloudComputeReporting/Playbooks/Prisma_Cloud_Compute_Vulnerability_and_Compliance_Reporting_README.md

```diff
@@ -0,0 +1,31 @@
+This playbook creates a summary report of the vulnerabilities and compliance issues found within a Prisma Cloud Compute deployment. The reports are generated in html format and can therefore be sent via email and other methods.
+
+## Dependencies
+This playbook uses the following sub-playbooks, integrations, and scripts.
+
+### Sub-playbooks
+This playbook does not use any sub-playbooks.
+
+### Integrations
+* PaloAltoNetworks_PrismaCloudCompute
```

Comment:
@nericksen I can't find the integration files in this PR

Reply:
removed